fairroot/base/sink/FairRootFileSink.cxx

```diff
@@ -200,13 +207,6 @@ void FairRootFileSink::TruncateBranchNames(TBranch* b, TString ffn)
     delete BrIter;
 }
 
-void FairRootFileSink::Close()
-{
-    if (fRootFile) {
-        fRootFile->Close();
-    }
-}
-
```

Comment:
But I think, this is public API still?

Reply:
Yes, without the declaration here, `FairRootFileSink` still have Close public API, which falls back to the implementation of its base class.